src/Event/Productions/ProductionCommandHandler.php

```diff
@@ -52,6 +61,45 @@ public function handleRemoveEventFromProduction(RemoveEventFromProduction $comma
     public function handleMergeProductions(MergeProductions $command): void
     {
         $toProduction = $this->productionRepository->find($command->getTo());
+        $fromProduction = $this->productionRepository->find($command->getFrom());
+
         $this->productionRepository->moveEvents($command->getFrom(), $toProduction);
+
+        $this->markNewMergedPairsAsLinked(
+            $fromProduction,
+            $toProduction
+        );
+    }
+
+    public function handleSkipEvents(SkipEvents $command): void
+    {
+        $this->similaritiesClient->excludePermanently(EventPair::fromArray($command->getEventIds()));
```

Comment:
I'm not sure if skipping suggestions should be a command. It's not really part of the core productions functionality inside the UDB3 backend but more of a necessity to make an external system work. Maybe this could just go directly inside the controller for the `skip` endpoint?

Reply:
@jdrieghe what is your take on this?

Reply:
Agreed. I think I suggested earlier to have a skip endpoint that automatically returns the next pair, making the permanent exclusion from GCP a part of that implementation because otherwise, we would get the same pair again.

Reply:
I don't see it as something that's not part of the core functionality of Productions. Whatever system we would use, internal or external to provide suggestions we would need a way of "rejecting" certain suggestions. 

But, regardless of that, I would still keep it in command for architectural consistency. If we are using the command pattern I would expect that all Use Cases/ Actions in an app are represented with a command

Reply:
I like your suggestion of the word "reject". We should model it in the domain as `RejectSuggestedEventPair($eventPair)`